src/manifolds/MetricManifold.jl

```diff
@@ -179,12 +302,16 @@ flat(::MetricManifold, ::Any...)
 end
 
 @doc raw"""
-    inverse_local_metric(M::AbstractcManifold, p, B::AbstractBasis)
+    inverse_local_metric(M::AbstractcManifold{𝔽}, p, B::AbstractBasis)
 
-Return the local matrix representation of the inverse metric (cometric) tensor, usually
-written ``g^{ij}``.
+Return the local matrix representation of the inverse metric (cometric) tensor
+of the tangent space at `p` on the [`AbstractManifold`](@ref) `M` with respect
+to the [`AbstractBasis`](@ref) basis `B`.
 
-See also [`local_metric`](@ref)
+The metric tensor (see [`local_metric`](@ref)) os isually denoted by ``G = (g_{ij}) ∈ 𝔽^{d×d}``,
```

Comment:
```suggestion
The metric tensor (see [`local_metric`](@ref)) is usually denoted by ``G = (g_{ij}) ∈ 𝔽^{d×d}``,
```
Is there actually a case when the metric tensor wouldn't have all real coefficients?

Reply:
For a `d` dimensional complex manifold and `d` (real valued) basis vectors, the coefficients would be complex. They would be real if we represent them as `2d` entries, but otherwise complex, I think.

Reply:
Yes, that's probably right.